express/python/opacity.py

```diff
@@ -60,6 +60,23 @@ def calculate_opacities(feature_row):
 	opacities = [min_opac if val==0.0 else round((val*0.95/max_exp + min_opac), 2) for val in exp_values]	
 	return opacities	
 
+def binary_search(feature, features):
+	L = 0
+	R = len(features) - 1
+	if (feature == features[L]):
+		return L
+	elif (feature == features[R]):
+		return R
+	while R - L != 1:
+		M = round(float((L+R)/2))
```

Comment:
Simply perform integer division with `(L + R) // 2`. Conversion to float is extraneous.